user/languages/python.md

```diff
@@ -102,7 +102,7 @@ python:
   - "3.5"
   - "3.6"
   # PyPy versions
-  - "pypy2.7"
+  - "pypy"
```

Comment:
We could go even further and remove it altogether.
And then remove it from above:

> To test your project against PyPy, add "pypy" or "pypy3" to the list of Pythons
 in your `.travis.yml`:

But it will look a bit strange that we have "2.6" listed (which is almost impossible to use on Travis now because Python devs have removed 2.6 support from critical packages) and not "pypy".

Reply:
So we could just remove 2.6 and pypy2.7 ?